### DISCUSSION TITLE
My css file can't be displayed on a live web in GitHub

### DISCUSSION THREAD

**Discussion by progressive-newbie263** [community/community#108296] [New to GitHub]
So i watched the guy name SuperSimpleDev on Youtube, and i recreate his youtube clone design (Only css and html atm, i will start to learn java soon after it) . But when i tried to launch it on GitHub, for some reason the css can not be applied to the html code, even though i was able to do it successfully through live server in vs code. Can anyone take a look on it and give me some advices on how to do it ? Been struggling on it for a while now after 2 hours of deleting , adding again so i'd be really grateful to be taught where did i go wrong

Here is the link of my recreated youtube clone  : https://github.com/progressive-newbie263/YT-clone-demo

**Comment by kyuna0312** [comment]
i think you should check path and then again publish. and pls make this answer xD

**Reply by progressive-newbie263** [reply]
i can't, Miss. I tried to find where i was wrong in the last 4 hours, but the result is just like this. One was from my live server vs code while the other is from github live server 

![Screenshot 2024-02-14 222959](https://github.com/community/community/assets/159697999/ccf15a6d-699f-454f-a73b-cd06e33fa1eb)

![Screenshot 2024-02-14 222834](https://github.com/community/community/assets/159697999/c9c38630-781a-4f35-8e66-ee5b53568ec6)
